nio/responses.py

```diff
@@ -479,6 +485,20 @@ def from_dict(cls, parsed_dict):
         )
 
 
+@attr.s
+class LogoutResponse(Response):
+    def __str__(self):
+        # type: () -> str
+        return "Logged out"
+
+    @classmethod
+    @verify(Schemas.empty, LogoutError)
+    def from_dict(cls, parsed_dict):
+        # type: () -> Union[LogoutResponse, ErrorResponse]
```

Comment:
This type annotation should have Dict[Any, Any] inside the parentheses.

Reply:
hm, just learned about Mypy: that was the thingy needed for typesetting checks.

But I can only partly comprehend the error/warning output of mypy to my problems. Probably my virtualenv misses stuff for letting it work error free...